packages/core/src/layout/ContentHeader/ContentHeader.tsx

```diff
@@ -91,7 +97,11 @@ export const ContentHeader: FC<ContentHeaderProps> = ({
   return (
     <Fragment>
       <Helmet title={title} />
-      <div className={classes.container}>
+      <div
+        className={classNames(classes.container, {
```

Comment:
A more MUI way to do this would be to make styles conditional based on props, docs here: https://material-ui.com/styles/basics/#adapting-based-on-props

End up with a bit less code as well and more focused place where styling and styling conditions happend.